### PR TITLE
fix(btc_server): remove pegin-conf cli flag

### PR DESCRIPTION
**Pull request by rwlockbg**
Ticket: https://github.com/botanix-labs/botanix/issues/579

### CODE REVIEW

File: bin/btc-server/src/util.rs
Comment (arminsabouri) [inline bin/btc-server/src/util.rs]: I would panic on everything execpt regtest  and signet.

Comment (arminsabouri) [reply bin/btc-server/src/util.rs]: confirmation depth for our testnet should be 1

**Review by arminsabouri — CHANGES_REQUESTED** [review]

File: bin/btc-server/src/util.rs
Comment (rwlockbg) [reply bin/btc-server/src/util.rs]: fixed

File: bin/btc-server/src/util.rs
Comment (arminsabouri) [inline bin/btc-server/src/util.rs]: I think we need to include regtest here. Otherwise our local set ups and int. should fail

Comment (arminsabouri) [reply bin/btc-server/src/util.rs]: Added this

**Review by arminsabouri — APPROVED** [review]

File: bin/btc-server/src/util.rs
Comment (rwlockbg) [reply bin/btc-server/src/util.rs]: thx

**Review by arminsabouri — APPROVED** [review]
@rwlockbg force pushed a rebase and dropped the merge commit. Waiting for CI to pass before merging

**Pull request merged**
